### PR TITLE
version bumps for 4.28 stream: eclipse-platform.common

### PR DESCRIPTION
**Pull request by gireeshpunathil**
Version bumps for 4.28 stream in the platform-common repo

Fixes: https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/issues/917

### CODE REVIEW

File: bundles/org.eclipse.jdt.doc.user/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.jdt.doc.user; singleton:=true
-Bundle-Version: 3.15.1700.qualifier
+Bundle-Version: 3.15.1800.qualifier
```
Comment (gireeshpunathil) [inline bundles/org.eclipse.jdt.doc.user/META-INF/MANIFEST.MF:5]: this is an unlisted change, but looked to me as a valid change, so included. pls let me know if that is not the case.

Comment (sravanlakkimsetti) [reply bundles/org.eclipse.jdt.doc.user/META-INF/MANIFEST.MF:5]: This may not be required now but will be required by M1. Its good

Comment (sravanlakkimsetti) [reply bundles/org.eclipse.jdt.doc.user/META-INF/MANIFEST.MF:5]: Normally pde environment suggests the version increments when we are updating a plugin. that is when we update the versions.

**Pull request merged**